### ISSUE TITLE
[Feature request] Add shortcuts for moving up/down the graph

### ISSUE THREAD

**Issue by stefan-van-de-griendt-quandago** [ForkIssues/TrackerWin#1829]
This comes from #333 and I figured creating a separate, isolated issue could help keeping it all clean and easier to manage.

I still can't find a shortcut for this, but it's really common for us to have multiple active branches. So, navigating manually by following a line in the graph is really inconvenient. Would love to have Ctrl + Up and Ctrl + Down for the most common scenarios, which is moving in the same branch. There could be some edge cases, such as

- `Ctrl + Down` into the originating branch, because this is the first commit of the current branch and there's only one parent;
- `Ctrl + Up` into the target branch, because this is the last commit of the current branch and there's only one child;
- `Ctrl + Down` when there are multiple parents, but there's no parent commit on the same branch, then nothing happens, else goes to the previous commit on the current branch;
- `Ctrl + Up` when there are multiple children, but there's no child commit on the same branch, then nothing happens, else goes to the next commit on the current branch.

**Comment by bilbothebaggins** [comment]
Add me to this.
I find Fork to be a great tool to navigate our messed up branch history (see eg: https://imgur.com/a/cyfQXxm)
But as you say:
* Navigating to a previous or next commit in a given branch can be a pain - as far as I can tell, there not only is no keyboard shortcut, there is no way at all to "jump" to a previous commit on a branch.

While I'm at it: Having keyboard shortcuts for pin/show/hide on a given branch would be very nice too.

cheers.

**Comment by bilbothebaggins** [comment]
I would like to mention that _GitExtensions_ has this feature (in the "Navigate" submenu of https://git-extensions-documentation.readthedocs.io/en/release-4.1/browse_repository.html#revision-graph) as shortcuts `CTRL+N` and `CTRL+P` and it is extremely valuable when jumping btw. (far apart) commits.

Would be really neat to get this.

**Comment by datgame** [comment]
Agree. I use the PARENTS: links on a commit to go down the tree, but then there's no back-button or CHILDREN-button.
